lib/Gitlab/Api/Environments.php

```diff
@@ -0,0 +1,43 @@
+<?php namespace Gitlab\Api;
+
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class Environments extends AbstractApi
+{
+    /**
+     * @param int $project_id
+     * @return mixed
+     */
+    public function all($project_id)
+    {
+        return $this->get($this->getProjectPath($project_id, 'environments'));
+    }
+
+    /**
+     * @param int $project_id
+     * @param array $parameters
```

Comment:
Could you add details about this argument like in:
https://github.com/m4tthumphrey/php-gitlab-api/blob/35ecf9b82984beace5eaea0df4fcbe4f77e2c285/lib/Gitlab/Api/Repositories.php#L179-L194

Reply:
Sure thing. Done, using github API description for the details.